### PR TITLE
Remove chart to harvester/charts repo

### PR DESCRIPTION
**Pull request by yaocw2020**
related pr: https://github.com/harvester/charts/pull/2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,12 @@ Users can deploy Harvester Load Balancer in any other Kubernetes clusters withou
 ## Deploying
 ```
 # helm v3
-helm install harvester-load-balancer deploy/harvester-load-balancer -n harvester-system --set kube-vip.interface=<interface name> --set kube-vip.address=<IP address>
+$ helm repo add harvester https://charts.harvesterhci.io
+$ helm repo add kube-vip https://kube-vip.io/helm-charts
+$ helm repo update
+$ helm install harvester/harvester-load-balancer:<version>
```
Comment (gitlawr) [inline README.md:16]: Is it still required to set those values? `-n harvester-system --set kube-vip.interface=<interface name> --set kube-vip.address=<IP address>`

**Review by gitlawr — DISMISSED** [review]
~~LGTM~~

**Review by gitlawr — COMMENTED** [review]
^^ mistaken approval

File: README.md (hunk 1 above)
Comment (yaocw2020) [reply README.md:16]: Oh, yes, interface is still required.

Comment (gitlawr) [reply README.md:16]: Then we should keep it here

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by guangbochen — APPROVED** [review]

**Pull request merged**